foursight_core/s3_connection.py

```diff
@@ -41,14 +41,21 @@ def put_object(self, key, value):
         else:
             return key, value
 
+    def get_all_objects(self):
+        # In fact, the parent does not implement this and will raise NotImplementedError,
+        # but adding this definition reminds us that that we didn't accidentally not implement this.
+        # (We didn't, did we?)  -kmp 7-Jun-2022
+        return super().get_all_objects()
```

Comment:
Not sure what utility this method even has, as the checks in S3 at least will grow unbounded (since we do not routinely clean them). You could not "get all of them" realistically even if you wanted to.